easybuild/easyconfigs/m/manta/manta-1.6.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
+
+name = 'manta'
+version = '1.6.0'
+
+homepage = 'https://github.com/Illumina/manta'
+description = """
+Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.
+It is optimized for analysis of germline variation in small sets of individuals and
+somatic variation in tumor/normal sample pairs. Manta discovers, assembles and
+scores large-scale SVs, medium-sized indels and large insertions within a
+single efficient workflow.
+"""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+github_account = 'Illumina'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['76ce7c4a95a3b92cb16381f9a3b091afa234948ae6f028982abbd49e21206421']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('Doxygen', '1.8.20'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Boost', '1.74.0'),
+    ('zlib', '1.2.11'),
+]
+
+# making included pyflow compatible with python3
+preinstallopts = "sed -i 's/0755/0o755/g' ./redist/pyflow-*/src/pyflow.py && "
+preinstallopts += "sed -i 's/DataDirException, e/DataDirException(e)/g' ./redist/pyflow-*/src/pyflow.py && "
+
+local_binaries = ['CheckSVLoci', 'DumpSVLoci', 'EstimateSVLoci', 'GenerateSVCandidates', 'GetAlignmentStats',
+                  'GetChromDepth', 'MergeAlignmentStats', 'MergeSVLoci', 'SummarizeAlignmentStats', 'SummarizeSVLoci',
+                  'TestAssembler']
+
+sanity_check_paths = {
+    'files': [
+        ['bin/%s' % x for x in ['configManta.py', 'runMantaWorkflowDemo.py']] +
+        ['libexec/%s' % x for x in ['denovo_scoring.py', 'extractSmallIndelCandidates.py']],
+    ],
+    'dirs': ['libexec'],
+}
+
+modextrapaths = {
+    'PATH': 'libexec',
+    'PYTHONPATH': 'lib/python',
+}
+
```

Comment:
@deniskristak can you add sanity check commands for one or two of the binaries and python scripts?